morpheus/cli.py

```diff
@@ -217,6 +217,12 @@ def tools(ctx: click.Context, **kwargs):
 @prepare_command()
 def onnx_to_trt(ctx: click.Context, **kwargs):
 
+    try:
+        import tensorrt
+    except:
+        logger.error("onnx-to-trt requires the tensorrt module and runtime to be installed.")
+        return
+
```

Comment:
Can we actually move this import check into the `onnx_to_trt.py` file? This check should really be done there and in case the utility is used outside of the CLI. Whis would allow us to remove the existing `import tensorrt as trt` that is in the `gen_engine` function and replace it with this code at the top level.